src/lib/chat/matrix-client.ts

```diff
@@ -238,12 +248,36 @@ export class MatrixClient implements IChatClient {
     }
 
     const result = await this.matrix.createRoom(options);
+
     // Any room is only set as a DM based on a single user. We'll use the first one.
     await setAsDM(this.matrix, result.room_id, users[0].matrixId);
 
+    await this.sendAdminMessage(result.room_id, users[0].userId, currentUserId, adminMessageType);
+
     return this.mapConversation(this.matrix.getRoom(result.room_id));
   }
 
+  private async sendAdminMessage(
+    roomId: string,
+    inviterId: string,
+    currentUserId: string,
+    type: AdminMessageType = AdminMessageType.CONVERSATION_STARTED
+  ) {
+    let content: any = {
+      type,
+      body: 'You have joined a conversation.',
+    };
+
+    if (type === AdminMessageType.JOINED_ZERO) {
+      content.inviteeId = currentUserId;
+      content.inviterId = inviterId;
+    } else if (type === AdminMessageType.CONVERSATION_STARTED) {
```

Comment:
We shouldn't be creating this type of event. We can already get this one from the standard Matrix events.